eventuate-tram-messaging-proxy-service/src/main/java/io/eventuate/tram/messaging/proxy/service/ProxyServiceMain.java

```diff
@@ -1,11 +1,12 @@
 package io.eventuate.tram.messaging.proxy.service;
 
+import io.eventuate.tram.messaging.proxy.consumer.EventuateMessageSubscriberConfiguration;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Import;
 
 @SpringBootApplication
-@Import(ProxyConfiguration.class)
+@Import(EventuateMessageSubscriberConfiguration.class)
```

Comment:
Where is `ProxyConfiguration` used if not here?

Reply:
EventuateMessageSubscriberConfiguration includes it.

https://github.com/eventuate-tram/eventuate-tram-messaging-http/blob/master/eventuate-tram-messaging-proxy-service/src/main/java/io/eventuate/tram/messaging/proxy/consumer/EventuateMessageSubscriberConfiguration.java#L12